src/lib/actions/reviews/review.action.ts

```
@@ -8,6 +8,9 @@ export async function createReviewAction(data: ReviewInput) {
     // Get authentication token
     const token = await getDecodeToken();
 
+    console.log("get token",token);
```

ESLint

[error] Unexpected console statement.

(no-console)